go/inst/rollback.go

```diff
@@ -0,0 +1,47 @@
+package inst
+
+// Main use case:
+// 1. User moves a group of slaves
+// 2. This hangs because of an unrelated issue
+// 3. User presses ^C to sent SIGINT
+// 4. Orchestrator tries to cleanup: e.g. start slaves.
+
+import (
+	"github.com/openark/golib/log"
+)
+
+type rollbackAction struct {
+	action   string
+	argument string
+}
+
+var rollbackActions []rollbackAction
+
+func AddRollbackAction(action string, argument string) {
+	act := rollbackAction{action, argument}
+	rollbackActions = append(rollbackActions, act)
+}
+
+func Rollback() {
```

Comment:
please make this function return an `error`. I realize it is unlikely we would check the return value calling this function. Nonetheless perhaps in some future testing scenario we would.